collect_app/src/test/java/org/odk/collect/android/mainmenu/MainMenuActivityTest.kt

```diff
@@ -430,4 +444,13 @@ class MainMenuActivityTest {
             assertThat(dialog, equalTo(null))
         }
     }
+
+    @Test
+    fun `when there has been a crash, opens CrashHandlerActivity and finishes`() {
+        CrashHandler.getInstance(application)!!.registerCrash(application, IllegalStateException())
+
+        val activity = Robolectric.setupActivity(MainMenuActivity::class.java)
+        assertThat(shadowOf(activity).nextStartedActivity, notNullValue())
```

Comment:
Shouldn't we also check that the new activity is CrashHandlerActivity as the title says? Here you just check if it's not null.

Reply:
Whoops! Yes absolutely.